### ISSUE TITLE
Many Hands Evaluated Wrong w/Wildcards

### ISSUE THREAD

**Issue by woodydeck** [gsdriver/poker-ranking#1]
Here is a test hand of a full house read three of a kind:

`console.log(evaluator.evaluateAndFindCards(['6h','6s','5s','Ts','Td'], { wildCards: ['5'], cardsToEvaluate:5}))`

`{ match: 'Three of a Kind', cards: [ '5s', '6h', '6s' ] }`

I have tested it and found a lot of misevaluated hands frequently with straights. It doesn't see them with wildcards half the time.

Edit: I think there's something weird in the nest of evaluateInternal, but it's hard to parse.

**Comment by gsdriver** [comment]
Actually the problem is that you should use "10s" rather than "Ts". If you make this change, then the code works properly. However, the code should reject with an "invalid card" error rather than effectively remove those cards and evaluate only the valid cards in the hand. I'm going to close this issue and create a new one to address this problem.
